src/txn/txn_rollback_to_stable.c

```diff
@@ -824,26 +827,40 @@ __rollback_abort_col_var(WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t r
             WT_RET(__rollback_abort_insert_list(
               session, page, ins, rollback_timestamp, &stable_update_found));
 
-        if (!stable_update_found && page->dsk != NULL) {
+        if (page->dsk != NULL) {
+            /* Unpack the cell. We need its RLE count whether or not we're going to iterate it. */
             kcell = WT_COL_PTR(page, cip);
             __wt_cell_unpack_kv(session, page->dsk, kcell, &unpack);
             rle = __wt_cell_rle(&unpack);
-            if (unpack.type != WT_CELL_DEL) {
+
+            /*
+             * If we found a stable update on the insert list, the on-disk value must be older than
+             * that update, so it too is stable(*) and any keys in the cell that _don't_ have stable
+             * updates don't need further attention. Then also, if the cell is deleted it must be
```

Comment:
If there are keys 10-15 and 10 and 12 have a stable update, keys 11, 14, and 15 do not have stable updates but also don't need attention. I agree, that comment is not super clear, but I'm not sure how to improve it. (it's already been around a few times)
Unstable updates are irrelevant here because they just got pruned out of the insert list. (I think. But I'm pretty sure.)

Reply:
Understood. How about adding something like `_don't_ have stable updates in the update list`?

Reply:
How about adding `update list` as `_don't_ have stable updates in the update list don't need further attention`?

Reply:
Did that and also expanded it some more; have a look at the latest version...